iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeLoginView.swift

```diff
@@ -78,6 +100,8 @@ struct NativeLoginView: View {
                     
                     if isAuthenticating {
                         ProgressView()
+                    } else {
+                        Spacer().frame(width: 20, height: 20)
```

Comment:
I wasn't sure if `dev` did this, but the progress view was making the layout "overflow" the rectangle background.  So I added a little more room to reserve its space.